src/strategies/property_readers.sml

```diff
@@ -169,8 +75,14 @@ fun dimension str =
     in
         List.foldr (fn (a, b) => a @ b) [] (dimensionsSplitOut @ dimensionsNoLabels)
     end;
+(*)
```

Comment:
Why is this commented out? If it is potentially useful in future, uncomment it but leave a comment as to why it is there. If it is a relic of old ideas, please delete it.

Reply:
Done